source/promise-value.d.ts

```diff
@@ -15,6 +15,12 @@ let data: Data = await asyncData;
 // Here's an example that shows how this type reacts to non-Promise types.
```

Comment:
Can you mention "nested" support on line 2?

Reply:
please see the latest change. It is on line 3.